src/components/custom/_signup-affiliate-details.js

```diff
@@ -317,22 +313,25 @@ const SignupAffiliateDetails = ({ autofocus, handleLogin, showModal }) => {
                                     )}
                                 ></AgreementLabel>
                                 <AgreementLabel
-                                    is_affiliate
                                     is_checked={is_terms_checked}
                                     handleChangeCheckbox={handleTermsChange}
-                                    link_text={localize(
-                                        'I have read and accepted Deriv’s <0>General business terms</0> and ',
-                                    )}
                                 >
-                                    <StyledLocalizedLink
+                                    {localize('I have read and accepted Deriv’s')}
+                                    <StyledLink
+                                        external="true"
+                                        to="/terms-and-conditions/#business-partners"
+                                        target="_blank"
+                                    >
+                                        {localize(' General business terms ')}
+                                    </StyledLink>
+                                    {localize('and')}
+                                    <StyledLink
                                         external="true"
                                         to="/tnc/business-partners-affiliates-and-introducing-brokers.pdf"
                                         target="_blank"
-                                        rel="noopener noreferrer"
-                                        secondary
                                     >
-                                        Affiliates and introducing brokers’ terms.
-                                    </StyledLocalizedLink>
+                                        {localize(' Affiliates and introducing brokers’ terms.')}
```

Comment:
The space shouldn't be inside the `localize`. Add a `{' '}` before if needed.

Reply:
Noted. This part of code was refactored, so no need in this.